src/service/fixed-layer.js

```diff
@@ -324,6 +310,44 @@ export class FixedLayer {
   }
 
   /**
+   * Calls `setupFixedSelectors` in a try-catch.
+   * Fails quietly with a dev error if call fails.
+   * This method should not be inlined to prevent TryCatch deoptimization.
+   * NoInline keyword at the end of function name also prevents Closure compiler
+   * from inlining the function.
+   * @param {!Array<string>} fixedSelectors
+   * @private
+   */
+  trySetupFixedSelectorsNoInline(fixedSelectors) {
```

Comment:
There is a [Closure bug](https://github.com/google/closure-compiler/issues/230) to consider Try-Catch before inlining but looks like it won't be fixed since V8 will have a fix for this deoptimization soon. I have asked V8 what version of V8/Chrome is expected to have the (hopefully final) [fix](https://codereview.chromium.org/1996373002).